extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -151,14 +188,20 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
 
     this.shoppingTagEl_ = this.renderShoppingTagTemplate_();
     this.onRtlStateUpdate_(this.storeService_.get(StateProperty.RTL_STATE));
-    this.mutateElement(() => {
-      createShadowRootWithStyle(
-        this.element,
-        this.shoppingTagEl_,
-        shoppingTagCSS
-      );
-      this.hasAppendedInnerShoppingTagEl_ = true;
-    });
+
+    this.measureMutateElement(
+      () => {
+        createShadowRootWithStyle(
```

Comment:
You're creating DOM, not measurements from the DOM, this doesn't need to be in a `measureElement`

Reply:
Addressed in https://github.com/ampproject/amphtml/pull/37704